postgres-appliance/scripts/post_init.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+set -Eeo pipefail
```

Comment:
Please remove it.
There are places where commands could exit with non-zero exit code (it is expected). We don't want the script to break because of that.